library/cwm/src/modules/CWM.rb

```diff
@@ -397,7 +397,7 @@ def cleanupWidgets(widgets)
       Builtins.foreach(widgets) do |w|
         @processed_widget = deep_copy(w)
         toEval = Convert.convert(
-          Ops.get(w, "clean_up"),
+          Ops.get(w, "cleanup"),
```

Comment:
Please put the clean_up clean-up to a separate commit with an explanation why this change is needed.